packages/tenant-readmodel-writer/test/consumerService.integration.test.ts

```diff
@@ -0,0 +1,264 @@
+/* eslint-disable functional/no-let */
+/* eslint-disable functional/immutable-data */
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
can you check if these are actually required?

Reply:
Removed